test/utils.ts

```diff
@@ -11,3 +11,9 @@ export function repeat(func: () => void, count: number) {
 
   return results;
 }
+
+export async function executionTime(func: () => Promise<any>): Promise<number> {
```

Comment:
Avoid `any`. Either use generics, or `never` type if you don't use the return type.